content/news/2020-08-28-gsoc-ends-2020.md

```diff
@@ -51,6 +74,17 @@ A quick overview of the Matrix protocol for reference: a client sends events to
  - [adding small needed features to the macros](https://github.com/ruma/ruma/pull/213)
  - [good ol' fashioned closing issues!](https://github.com/ruma/ruma/pull/234)
 
- One of my personal goals was to become more familiar with git. With the help of my mentor I now feel more confident using this tool that is so essential to developers. I became fairly adept at merging/rebasing, and navigating all the headaches that come with that. I learned plenty of new commands, a few highlights, `cherry-pick` and specific uses of `reset` to avoid copy/pasting fixes and adding more commits. I used the `reset` command to craft good commits, splitting work into appropriate chunks. I am glad that I had the opportunity to hone my git skills. I feel like I have accomplished my goal and then some!
+ One of my personal goals was to become more familiar with git. With the help
+ of my mentor I now feel more confident using this tool that is so essential to
+ developers. I became fairly adept at merging, rebasing, and navigating all the
+ headaches that come with that. I learned plenty of new commands. A few
+ highlights: `cherry-pick` and specific uses of `reset` to avoid copy-pasting
+ fixes and adding more commits. I used the `reset` command to craft good
+ commits, splitting work into appropriate chunks. I am glad that I had the
+ opportunity to hone my git skills. I feel like I have accomplished my goal and
+ then some!
 
- I am proud of the work that I have done. Being part of moving ruma much closer to the `0.22` release. Creating macros to generate types specific to the Matrix specification. Working with the community that has grown around ruma has been rewarding and I plan on sticking around to continue working.
\ No newline at end of file
+I am proud of the work that I have done: Being part of moving ruma-events much
+closer to the `0.22` release and creating macros to generate types specific to
+the Matrix specification. Working with the community that has grown around Ruma
+has been rewarding and I plan on sticking around to continue working.
```

Comment:
This looks kind of weird. Maybe just leave out the last part? Or some small rewrite?

```suggestion
has been rewarding and I plan on sticking around.
```

Reply:
I like it without the end, I was making sure the point was made, clearly it was.